Client/Assets/NetError.css

```diff
@@ -10,7 +10,7 @@ body {
 }
 
 body {
-    background-color: white;
+    background-color: #F8F8F6;
```

Comment:
I'm not sure if it would be possible, out of curiosity, would it be possible to reference here one of the semantic colors instead of hard coding the value? :thinking: